WordPress/gutenberg
# Add the stack block variation to the group block

## What?

This PR allows switching the "flex" layout orientation in the "row" block. It also adds a new "stack" group block variation for the vertical "flex" layout. 

## Why?

The "row" name means something horizontal I think so having a dedicated "stack" variation might remove some confusion for users.

https://github.com/WordPress/gutenberg/issues/39647#issuecomment-1076238706

**Note** Right now all the variations (group, row and stack) share the same icon, I'm happy to change it if needed. Happy to tweak the copy of the descriptions as well.

## Testing Instructions

1- Insert a row block
2- Switch the "orientation" of the layout to "vertical"
3- Notice that the variation name is now "stack".
